gui/wxpython/datacatalog/tree.py

```diff
@@ -595,6 +622,17 @@ def OnGetItemImage(self, index, which=wx.TreeItemIcon_Normal, column=0):
         except ValueError:
             return 0
 
+    def OnGetItemTextColour(self, index):
+        """Overriden method to return colour for each item.
+           Used to distinquish lock and ownership on mapsets."""
+        node = self._model.GetNodeByIndex(index)
+        text_colour = self.GetTextColour()
+        if node.data['lock'] == True or node.data['user']:
+            text_colour.SetTextColour(wx.SYS_COLOUR_GRAYTEXT)
+        else:
+            text_colour.SetTextColour(wx.SYS_COLOUR_WINDOWTEXT)
```

Comment:
You need to return directly the wx.SYS....

Delete '== True '

Reply:
Edited.